src/emmy/env.cljc

```diff
@@ -97,7 +97,7 @@
                :refer
                (into [] (keys (ns-publics 'emmy.env)))]))
 
-(defmacro literal-function
+(u/sci-macro literal-function
```

Comment:
can the new `import-def` handle macros? If so I can delete these and move them down into the `import-vars` block below... so good!